deployments/azuredeploylogs.json

```diff
@@ -0,0 +1,307 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters":
+    {
+        "ContainerName":
+        {
+            "defaultValue": "logs",
+            "minLength": 3,
+            "maxLength": 63,
+            "type": "string",
+            "metadata":
+            {
+                "description": "The name of the blob container within the storage account."
+            }
+        },
+        "LogzioURL":
+        {
+            "defaultValue": "https://listener.logz.io:8071",
+            "type": "string",
+            "allowedValues":
+            [
+                "https://listener.logz.io:8071",
+                "https://listener-au.logz.io:8071",
+                "https://listener-ca.logz.io:8071",
+                "https://listener-eu.logz.io:8071",
+                "https://listener-nl.logz.io:8071",
+                "https://listener-uk.logz.io:8071",
+                "https://listener-wa.logz.io:8071"
+            ],
+            "metadata":
+            {
+                "description": "The Logz.io listener URL for your region."
```

Comment:
Please reach out to @nico-shishkin to go over the descriptions of that parameters